plugins/calibre.koplugin/wireless.lua

```diff
@@ -160,10 +163,6 @@ function CalibreWireless:JSONReceiveCallback(host, port)
             end
             this.connect_message = true
             UIManager:scheduleIn(1, this.password_check_callback)
-            if this.failed_connect_callback then
-                -- Don't disconnect if we connect in 10 seconds
```

Comment:
I'm not seeing anything that actually delays a connection anywhere, which is what I meant by "makes no sense" ^^.